internal/pkg/cli/recorder/recorder.go

```diff
@@ -127,6 +139,89 @@ func (r *Recorder) processData(mntns uint32) error {
 	return fmt.Errorf("find mntns %d in bpf data map", mntns)
 }
 
+func (r *Recorder) generateAppArmorProfile() apparmorprofileapi.AppArmorAbstract {
+	processed := r.bpfRecorder.GetAppArmorProcessed()
+
+	abstract := apparmorprofileapi.AppArmorAbstract{}
+	enabled := true
+	var gotExec bool
+
+	exec := apparmorprofileapi.AppArmorExecutablesRules{}
+	if len(processed.FileProcessed.AllowedExecutables) != 0 {
+		sort.Strings(processed.FileProcessed.AllowedExecutables)
+		exec.AllowedExecutables = &processed.FileProcessed.AllowedExecutables
```

Comment:
nit: you can use directly here the `abstract.Executable` and you can get rid of `gotExec` variable.